checkov/terraform/module_loading/loaders/bitbucket_access_token_loader.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from checkov.terraform.module_loading.loaders.git_loader import GenericGitLoader
+
+
+class BitbucketAccessTokenLoader(GenericGitLoader):
+    def discover(self):
+        self.username = os.getenv('BITBUCKET_USERNAME', '')
+        self.app_password = os.getenv('BITBUCKET_APP_PASSWORD', '')
+        self.token = os.getenv('BITBUCKET_TOKEN', '')
+
+    def _is_matching_loader(self) -> bool:
+        # https://www.terraform.io/docs/modules/sources.html#bitbucket
+        if self.token:
+            # https://support.atlassian.com/bitbucket-cloud/docs/use-oauth-on-bitbucket-cloud/
+            if self.module_source.startswith("bitbucket.org"):
```

Comment:
From the looks of it I feel like there's a lot of code duplication between Bitbucket, Github and Gitlab.

Isn't it possible to extract this logic to `GenericGitLoader._is_matching_loader` (to the parent class) and add the prefix 
(for example `bitbucket.org`) as the class property, and use it instead of the hardcoded string?